libs/shared/src/main/java/io/crate/concurrent/limits/ExpAvgMeasurement.java

```diff
@@ -0,0 +1,68 @@
+/**
+ * Copyright 2018 Netflix, Inc.
```

Comment:
Same here, maybe use ours instead? maybe I'm wrong, this license stuff always confuses me ;)

Reply:
No idea, felt fair to keep the original one as I hardly changed anything. Saw in opendistro plugins that they add multiple headers like this:

```
/*
 * Copyright 2015-2018 _floragunn_ GmbH
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an "AS IS" BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

/*
 * Portions Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the "License").
 * You may not use this file except in compliance with the License.
 * A copy of the License is located at
 *
 * http://www.apache.org/licenses/LICENSE-2.0
 *
 * or in the "license" file accompanying this file. This file is distributed
 * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
 * express or implied. See the License for the specific language governing
 * permissions and limitations under the License.
 */

```

Should we do that?

Reply:
ah ok, yeah then maybe lets do that. but no strong opinion, you could also leave as it.

Reply:
I'll leave it as is for now. We can change it once we make some more changes to the files.